### DISCUSSION TITLE
How to use other browser based on Chromium?

### DISCUSSION THREAD

**Discussion by mikeon** [c9fe/22120#114] [Q&A] [answered]
Hey, I want to ask if functionallity is already there. Is it possible to set in the settings or run "diskernet" with args such a way to run by default "Vivaldi", "Brave" or "Edge" browsers?

**Comment by crisdosaygo** [comment, marked as answer]
Hey @mikeon I'm sorry for my delay in replying you, I have been busy.

It's not possible with args just yet, but I want to enable this soon. Thank you for saying it here, it's a really useful feature.

I'm thinking something like:


`./diskernet <path-to-brower>`

or 

`/diskernet <browser-command>`

Where path-to-browser is the actualy path to the browser binary and browser-command is what you would type at the shell prompt if you wanted to open a binary from the command line. For example for google chrome on linux, you can type google-chrome-stable from the shell prompt, and its path may be something like /usr/bin/google-chrome-stable

**Reply by mpr1255** [reply]
Is this working now? Want to use with brave.

**Reply by crisdosaygo** [reply]
Hi @mpr1255 ! Thanks. You’re certainly welcome to open a PR or persuade a mate to. Should not be too hard. Updating the browser path to point to brave. Have a go! Bad idea? 🥹

**Reply by mpr1255** [reply]
Alright, I'll give it a go!

**Comment by crisdosaygo** [comment]
This is solved in latest @mikeon @mpr1255
